src/turbolinks/controller.coffee

```diff
@@ -217,6 +218,9 @@ class Turbolinks.Controller
     root = @getSetting("root") ? "/"
     new Turbolinks.Location root
 
+  isCacheable: ->
+    @getSetting("cache-control") != 'no-cache'
```

Comment:
:+1: Apart from style preferences, I'm just releasing that `isnt` translates to `!==`, and that's much better than `!=`.

Will change